server/backend/cron/movies.go

```diff
@@ -1,59 +1,37 @@
 package cron
 
 import (
-	"encoding/json"
-	"encoding/xml"
-	"errors"
-	"fmt"
-	"io"
-	"net/http"
-	"os"
 	"regexp"
-	"strings"
+	"slices"
 	"time"
 
+	"github.com/TUM-Dev/Campus-Backend/server/backend/cron/movie_parsers"
+
 	"github.com/guregu/null"
 
-	"github.com/PuerkitoBio/goquery"
 	"github.com/TUM-Dev/Campus-Backend/server/model"
 	log "github.com/sirupsen/logrus"
 )
 
-type MovieItems struct {
-	Title     string `xml:"title"`
-	Link      string `xml:"link"`
-	PubDate   string `xml:"pubDate"`
-	Location  string `xml:"location"`
-	Enclosure struct {
-		Url    string `xml:"url,attr"`
-		Length string `xml:"length,attr"`
-		Type   string `xml:"type,attr"`
-	} `xml:"enclosure"`
-}
-
-type MovieChannel struct {
-	Items []MovieItems `xml:"item"`
-}
-
 const (
 	MovieImageDirectory = "movie/"
 )
 
 func (c *CronService) movieCron() error {
 	log.Trace("parsing upcoming feed")
-	channels, err := parseUpcomingFeed()
+	var allMovieLinks []string
+	if err := c.db.Model(&model.Kino{}).Distinct().Pluck("Link", &allMovieLinks).Error; err != nil {
```

Comment:
The table "kino" doesn't have the column "Link" in my instance which leads to an error.

Reply:
![image](https://github.com/TUM-Dev/Campus-Backend/assets/26258709/df8158f3-e3df-4131-951c-0bfc8a7df1da)

Cannot reproduce this exact issue.
The column exists both on fresh re-running main and this branch.
I have fixed a bug though which lead to fresh rerunning failing => this might be a culprit.